packages/ckeditor5-dev-env/lib/release-tools/utils/cli.js

```diff
@@ -37,6 +38,73 @@ const cli = {
 			.then( answers => answers.confirm );
 	},
 
+	/**
+	 * Asks a user for a confirmation for publishing changes.
+	 *
+	 * @param {Map} packages Packages to release.
+	 * @returns {Promise}
+	 */
+	confirmPublishing( packages ) {
```

Comment:
It produces: 

![image](https://user-images.githubusercontent.com/2270764/43721664-b903758e-9993-11e8-8f05-9227ee0b485f.png)

Packages for publishing are highlighted, packages with no changes are darker.